lib/importexport/helpers.js

```diff
@@ -149,10 +159,11 @@ var _importZip = function(filePath, cb, retries){
         return cb(msg);
       }
 
-      parsers[name] = function(parserCallback){
-        importHandler[fileType](data, parserCallback);
-      };
-
+      parsers[name] = function(parserData) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)